src/server/auth.ts

```diff
@@ -6,8 +6,8 @@ import {
 } from "next-auth";
 import DiscordProvider from "next-auth/providers/discord";
 import { PrismaAdapter } from "@next-auth/prisma-adapter";
-import { env } from "~/env.mjs";
-import { prisma } from "~/server/db";
+import { prisma } from "@pb/server/db";
```

Comment:
Could you kindly ensure that there is a space between the various import types, please?